### PR TITLE
[net config] Added net config module

### PR DESCRIPTION
**Pull request by jprestwo**
- OCF and DGRAM can now use the same network/BLE config API's

Signed-off-by: James Prestwood <james.prestwood@intel.com>

### CODE REVIEW

File: src/zjs_net_config.c [hunk 1]
```diff
@@ -1,6 +1,49 @@
...
+static uint8_t ble_enabled = 0;
+#endif
 
-#include "zjs_ocf_ble.h"
+void zjs_net_config(void)
+{
+#if defined(CONFIG_NET_L2_BLUETOOTH)
+    if (!ble_enabled) {
+        zjs_init_ble_address();
+        if (bt_enable(NULL)) {
```
Comment (jimmy-huang) [inline src/zjs_net_config.c]: what is the return value of bt_enable here? if it not boolean maybe check if != error?

Comment (grgustaf) [inline src/zjs_net_config.c]: looks right, zero is success

**Comment by grgustaf** [comment]
+1

**Comment by pfalcon** [comment]
I'm as usual late with review...

This looks definitely good! For a moment I thought that maybe this is a JS-level module which allows to set local IP addresses, etc., but I see it's more an internal C module to consolidate related functions. Well, that's already a good step forwards, and I hope we'll get to JS-level module too eventually (one question is what to model its API on).

**Pull request merged**